src/libs/OptionsListUtils.ts

```diff
@@ -1333,20 +1333,20 @@ function getShareLogOptions(options: OptionList, betas: Beta[] = []): Options {
  * Build the IOUConfirmation options for showing the payee personalDetail
  */
 function getIOUConfirmationOptionsFromPayeePersonalDetail(personalDetail: OnyxEntry<PersonalDetails>, amountText?: string): PayeePersonalDetails {
-    const formattedLogin = LocalePhoneNumber.formatPhoneNumber(personalDetail?.login ?? '');
+    const login = personalDetail?.login ?? '';
     return {
-        text: PersonalDetailsUtils.getDisplayNameOrDefault(personalDetail, formattedLogin),
-        alternateText: formattedLogin || PersonalDetailsUtils.getDisplayNameOrDefault(personalDetail, '', false),
+        text: LocalePhoneNumber.formatPhoneNumber(PersonalDetailsUtils.getDisplayNameOrDefault(personalDetail, login)),
+        alternateText: LocalePhoneNumber.formatPhoneNumber(login || PersonalDetailsUtils.getDisplayNameOrDefault(personalDetail, '', false)),
         icons: [
             {
                 source: personalDetail?.avatar ?? FallbackAvatar,
-                name: personalDetail?.login ?? '',
+                name: LocalePhoneNumber.formatPhoneNumber(personalDetail?.login ?? ''),
```

Comment:
Is this change necessary here? Instead, we can simply use `LocalePhoneNumber.formatPhoneNumber` directly where the `icons.name` key is being used.

Reply:
same here, i did a test on spliting expenses and i agree. It will be good if you can also confirm by a visual test